### PR TITLE
PIM-3825: [BackendProcess] Change the family of products

### PR DESCRIPTION
**Pull request by grena**
- [x] Add missing specs

### CODE REVIEW

File: features/mass-action/change_status.feature [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-  I need to be able to change the status of many products at once
+  I need to be able to configure mass edit action via a form
 
   Background:
     Given the "apparel" catalog configuration
-    And I am logged in as "Julia"
 
-  Scenario: Enable many products at once
+  Scenario: Configure the operation to enable many products at once
+    And I am logged in as "Julia"
```
Comment (jmleroux) [inline features/mass-action/change_status.feature]: Why did you this step from background ?

File: src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php [hunk 2]
```diff
@@ -203,6 +207,11 @@ public function performAction($operationAlias)
             $jobInstance = $this->jobManager->getJobManager()
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: $this->jobManager->getJobManager() looks weird

File: src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php [hunk 3]
```diff
@@ -203,6 +207,11 @@ public function performAction($operationAlias)
...
                 ->getRepository('AkeneoBatchBundle:JobInstance')
                 ->findOneByCode($jobCode);
+
+            if (null === $jobInstance) {
+                throw new NotFoundResourceException(sprintf('No job found with job code "%s"', $jobCode));
+            }
+
             $jobInstance = $this->getJobInstance($jobInstance->getId());
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: I don't get this line :)

File: src/Pim/Bundle/EnrichBundle/MassEditAction/Handler/UpdateProductHandler.php [hunk 4]
```diff
@@ -64,24 +64,20 @@ public function __construct(
...
-        $action = $actions[0];
 
         foreach ($paginator as $productsPage) {
             foreach ($productsPage as $product) {
-                // TODO: Use this with the new version of productUpdater :
-//                foreach ($actions as $action) {
-//                    $this->productUpdater->setValue($product, $action['field'], $action['value']);
-//                }
+                foreach ($actions as $action) {
+                    $this->productUpdater->set($product, $action['field'], $action['value']);
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/MassEditAction/Handler/UpdateProductHandler.php]: setData when you'll rebase the PR

File: src/Pim/Bundle/EnrichBundle/MassEditAction/Handler/UpdateProductHandler.php [hunk 5]
```diff
@@ -64,24 +64,20 @@ public function __construct(
...
+                }
 
-                $product->setEnabled($action['value']);
-//                $product->setFamilyId($action['value']);
                 $this->stepExecution->incrementSummaryInfo('mass_edited');
+                // TODO: validation & skip
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/MassEditAction/Handler/UpdateProductHandler.php]: for validation, you can take a look on https://github.com/akeneo/pim-community-dev/blob/master/src/Pim/Bundle/CatalogBundle/Manager/ProductTemplateApplier.php#L75

**Pull request merged**